hw5/Task_7.ts

```diff
@@ -0,0 +1,4 @@
+//Task 7
+// Напишите самовызыввающиеся функцию подсчет факторила числа: число должно передаваться как параметр функции
```

Comment:
https://blog.mgechev.com/2012/08/29/self-invoking-functions-in-javascript-or-immediately-invoked-function-expression/
(about self invoking functions (last slide of the presentation))

Reply:
Though I checked the link you shared, I could't apply this function type in the given task.
Instead, I calculated factorial via generic function.

Reply:
Try something like this
<img width="579" alt="image" src="https://user-images.githubusercontent.com/47811445/218330461-53933bd2-7955-4e5a-8533-5d979968363f.png">

Reply:
Thanks. Added a function.